src/components/Card/Card.tsx

```diff
@@ -21,6 +21,9 @@ const Card = ({ title, image, id, sentiment, updateUserSentiment }: CardProps):
             <img className="article-image cy-article-image" src={image} alt={title} />
             <h2 className="article-title cy-article-title">{title}</h2>
           </Link>
+          {sentiment >= 7 && <h3 className="article-sentiment green">Sentiment rating: {sentiment}</h3>}
+          {sentiment < 4 && <h3 className="article-sentiment red">Sentiment rating: {sentiment}</h3>}
+          {(sentiment >= 5 && sentiment <= 6) && <h3 className="article-sentiment blue">Sentiment rating: {sentiment}</h3>}
```

Comment:
Interesting choice of blue here. I kind of like it representing neutral, since WellNews has blue themes and the goal of the app is balance.